src/app/core/authentication/authentication.interceptor.ts

```diff
@@ -36,7 +36,14 @@ export class AuthenticationInterceptor implements HttpInterceptor {
     if (this.settingsService.tenantIdentifier) {
       httpOptions.headers['Fineract-Platform-TenantId'] = this.settingsService.tenantIdentifier;
     }
-    request = request.clone({ setHeaders: httpOptions.headers });
+
+    /**
+     * This is made because we added apache superset and in superset we needed to send request to superset backend(localhost:8088) 
+     */
+    if (request.url.includes("fineract-provider")) {
```

Comment:
i dont see moving the `request = request.clone({ setHeaders: httpOptions.headers });` behind a condition what to do with apache superset...

Reply:
Actually, apache superset will run as an independent app so we didn't want to send "fineract-provider" as a header because it will not work otherwise. for embedding the apache superset we are not sending request to the fineract ,we are running the apache superset at the port :8088 and then sending the request to it, that's why added this